src/lib/y2partitioner/widgets/fstab_options.rb

```diff
@@ -66,7 +66,10 @@ def delete_fstab_option!(option)
 
       def add_fstab_options(*options)
         # The options can only be modified using BlkDevice#fstab_options=
-        filesystem.fstab_options = filesystem.fstab_options + options
+        # so we can't just append options with << or push
+        new_options = filesystem.fstab_options
+        options.each { |opt| new_options << opt }
+        filesystem.fstab_options = new_options
```

Comment:
To be honest, I did not understand why the previous version did not work.

Reply:
It didn't work if multiple options were passed as a function argument.

And since that filesystem.fstab_options is closely connected to those SWIG bindings, i.e. the underlying C++ object is a std::list<std::string>, not a real Ruby Array<String>, it doesn't provide the Ruby _Enum_ methods which makes it a bit awkward to handle.

Reply:
Wasn't that automatically handled by `Filesystem#fstab_options`? In theory it does internally basically what you are doing here.

Reply:
I tried, and I got another very weird error that boiled down to "no clue what to do with a `std::list<std::string>` IIRC.